### PR TITLE
Mocking Shipments

### PR DESCRIPTION
**Pull request by matzayonc**

### CODE REVIEW

File: migrations/fetch.ts
Comment (tommysr) [inline migrations/fetch.ts:1]: what's the purpose of this script?

**Review by tommysr — COMMENTED** [review]
ready to merge after explaining purpose of the fetch script

File: tests/sdk.ts
[the remark this answers is not in the document]
Comment (matzayonc) [reply tests/sdk.ts:1]: I think a prefered way would be to import it in test from the app but getting module errors

Moved to separate dir for now

File: migrations/fetch.ts
Comment (matzayonc) [reply migrations/fetch.ts:1]: as the name implies general fetching and reading state to confirm the data on the front-end

**Review by tommysr — APPROVED** [review]

**Pull request merged**